website/addons/dryad/model.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+
+from modularodm import fields
+
+from website.addons.dryad import utils as dryad_utils
+from website.addons.base import AddonUserSettingsBase, AddonNodeSettingsBase
+
+
+class AddonDryadUserSettings(AddonUserSettingsBase):
+    """
+     Stub in place for Dryad addon v2 upgrade.
+
+     As migrations are a little tricky, this stub is being left in since a user
+     settings class will be needed in the future.
+    """
+    has_auth = True
+
+
+class AddonDryadNodeSettings(AddonNodeSettingsBase):
+    """
+        A Dryad node is a collection of packages. Each package is specified by a DOI, and the title is saved automatically
+    """
+    dryad_package_doi = fields.StringField()
+    has_auth = True
+    provider_name = 'dryad'
+    connection = dryad_utils.DryadRepository()
```

Comment:
I like it. This might end up being replaced with an external provider, but for now, I wanted a proxy method for accessing the repository to make refactoring easier.